pkg/orchestration/wiring/wiringutil/svccatentangler/plugin.go

```diff
@@ -109,14 +110,18 @@ func (e *SvcCatEntangler) constructServiceInstance(resource *orch_v1.StateResour
 	}, nil
 }
 
-func (e *SvcCatEntangler) constructResourceContract(resource *orch_v1.StateResource, context *wiringplugin.WiringContext) (wiringplugin.ResourceContract, error) {
+func (e *SvcCatEntangler) constructResourceContract(resource *orch_v1.StateResource, smithResource smith_v1.Resource, context *wiringplugin.WiringContext) (wiringplugin.ResourceContract, error) {
 	// TODO(kopper): Actually implement.
 	return wiringplugin.ResourceContract{
-		Data: []wiringplugin.DataItem{
-			{
-				Name: "fakeNotEmptyContract",
-			},
+		Shapes: []wiringplugin.Shape{
+			knownshapes.NewBindableEnvironmentVariables(smithResource.Name),
+			knownshapes.NewBindableIamAccessible(smithResource.Name, "IamPolicySnippet"),
```

Comment:
path should be a parameter to `SvcCatEntangler` as it may be different.

Reply:
See comment about not providing IamAccessible - I don't think this util code should be aware of this shape at all.